app/helpers/react_on_rails_helper.rb

```diff
@@ -90,10 +90,18 @@ def define_render_if_dom_node_present(react_component, data_variable, dom_id, tr
     <<-JS.strip_heredoc
```

Comment:
Let's move this method over to lib/react_on_rails/react_renderer.rb.

Then we can reuse the common lines from 96 to 103.

The below code has the var names hard coded as `reactComponent` and `props`. The upper code uses the rails variables.

```javascript
        var element;

        if (Object.getPrototypeOf(reactComponent) === this.React.Component) {
          element = this.React.createElement(reactComponent, props);
        } else {
          // when using Redux, we need to pull the component off the wrapper function
          element = reactComponent(props);
        }
```